src/main/java/pro/belbix/ethparser/web3/Web3Service.java

```diff
@@ -408,6 +408,23 @@ public void subscribeTransactionFlowable() {
         log.info("Subscribe to Transaction Flowable");
     }
 
+    public void subscribeTransactionFlowableRange(Integer start, Integer end)
```

Comment:
add default values EARLIEST and LATEST from web3j lib
I know, EARLIEST will not use, but LATEST should be helpful